converter/shieldhit/beam.py

```diff
@@ -133,7 +133,7 @@ class BeamConfig:
 RNDSEED      	89736501     ! Random seed
 JPART0       	{particle}            ! Incident particle type{particle_optional_comment}
 {optional_heavy_ion_line}
-TMAX0      	{energy} {energy_spread}       ! Incident energy and energy spread; both in (MeV/nucl)
+TMAX0      	{energy} {energy_spread}       ! Incident energy and energy spread [MeV/nucl]
```

Comment:
see comment above for unit

`Incident energy and energy spread` -> `Mean and spread (1 sigma) of primary particle kinetic energy`